integration/testrunner/db.go

```diff
@@ -82,34 +104,44 @@ func makeBigQueryClient(ctx context.Context, tb testing.TB, projectID string) *b
 	return client
 }
 
-// This calls the database to check that an audit log exists. It uses the retries that are specified in the Config
-// file. This method assumes that only a single audit log will match, which constitutes success.
-func queryIfAuditLogExistsWithRetries(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *Config, testName string) {
-	tb.Helper()
-
-	queryIfAuditLogsExistWithRetries(ctx, tb, bqQuery, cfg, testName, 1)
-}
-
 // This calls the database to check that an audit log exists. It uses the retries that are specified in the Config
 // file. This method allows for specifying how many logs we expect to match, in order to handle streaming use cases.
-func queryIfAuditLogsExistWithRetries(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *Config, testName string, expectedNum int64) {
+func queryAuditLogsWithRetries(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *Config, testName string) []*logpb.LogEntry {
 	tb.Helper()
-
+	var logEntries []*logpb.LogEntry
 	b := retry.NewExponential(cfg.LogRoutingWait)
```

Comment:
We should use a different retry strategy. Thinking fixed backoff with 5 sec interval. The max retries should be around (5m / 5s) = 60

Reply:
done